src/main/java/com/bookstore/model/Book.java

```diff
@@ -0,0 +1,31 @@
+package com.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "book")
```

Comment:
table name should be plural
```suggestion
@Table(name = "books")
```